src/ORM/Exception/PersistenceFailedException.php

```diff
@@ -20,5 +20,5 @@
 class PersistenceFailedException extends Exception
 {
 
-    protected $_messageTemplate = 'Failed to %s an entity from %s with primary key "%s" because of: %s';
+    protected $_messageTemplate = 'Entity %s failure';
```

Comment:
Personally, I like to add full stops at the end of such messages. This way I know there is no ommited parameter at the end.